README.md

```diff
@@ -1,3 +1,54 @@
 # Mollie PHP Coding Standards
 
 Contains PHP coding standards like rules for PHP-CS-Fixer that serves for purpose of standardization.
+
+## Installation
+[Get Composer](https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md)
+
+```bash
+composer require --dev  mollie/php-coding-standards:v1.0
```

Comment:
I wouldn't do `:v1.0` but `^1.0` (with a space in between).

Even better is to just leave it away and let composer handle the correct versioning on install.